stdlib/tkinter/__init__.pyi

```diff
@@ -452,14 +481,14 @@ class CallWrapper:
     def __call__(self, *args): ...
 
 class XView:
-    def xview(self, *args): ...
-    def xview_moveto(self, fraction): ...
-    def xview_scroll(self, number, what): ...
+    def xview(self) -> Tuple[float, float]: ...
+    def xview_moveto(self, fraction: float) -> None: ...
+    def xview_scroll(self, number: int, what: Literal["units", "pages"]) -> None: ...
```

Comment:
Quoting the manual page:

>What must be  units,  pages
                     or  pixels. If what is units or pages then number must be
                     an integer, otherwise number may be specified in  any  of
                     the  forms  acceptable to Tk_GetPixels, such as “2.0c” or
                     “1i” (the result is rounded to the nearest integer value.
                     If  no  units  are given, pixels are assumed). 

We should reflect this with overloads:

```suggestion
    @overload
    def xview_scroll(self, number: int, what: Literal["units", "pages"]) -> None: ...
    @overload
    def xview_scroll(self, number: _ScreenUnits, what: Literal["pixels"] = ...) -> None: ...
```

Reply:
Good catch! I'll make that change (and in the YView class too).